pkg/volume/volume.go

```diff
@@ -132,11 +132,39 @@ type Attacher interface {
 	Attach() error
 }
 
```

Comment:
Good point. Actually both only make sense on kubelet. So it would make sense to move both to `PhysicalAttacher`